webhook_to_fedora_messaging/move.py

```diff
@@ -0,0 +1,145 @@
+import csv
+from asyncio import run
+from datetime import datetime
+from json import dumps
+from uuid import uuid4
+
+from sqlalchemy import select
+from sqlalchemy.exc import NoResultFound
+from sqlalchemy.orm import selectinload
+
+from webhook_to_fedora_messaging.config import get_config, set_config_file
+from webhook_to_fedora_messaging.database import get_session
+from webhook_to_fedora_messaging.models import Service, User
+from webhook_to_fedora_messaging.models.owners import owners_table
+
+
+userscsv_fileloca = "/home/archdesk/Projects/webhook-to-fedora-messaging/github2fedmsg-users.csv"
```

Comment:
Could you please use arguments for that? It's fine if the script is not as clean as the rest of the codebase, but this will literally not work outside of your computer.

Reply:
Not click-ifying it, but leaving it at 

```python
userscsv_fileloca = "PATH_TO_USER_DATABASE_EXPORT_CSV"
reposcsv_fileloca = "PATH_TO_REPO_DATABASE_EXPORT_CSV"
pairscsv_fileloca = "PATH_TO_PAIR_DATABASE_EXPORT_CSV"
```

Reply:
The lines with your personal path are still there, no?

Reply:
If you don't want to use click, there's still argparse, or even environment variables, which will be more practical than editing the source code to run the script.